src/opm/parser/eclipse/EclipseState/Grid/FieldPropsManager.cpp

```diff
@@ -109,6 +127,87 @@ std::size_t FieldPropsManager::active_size() const {
     return this->fp->active_size;
 }
 
+void FieldPropsManager::apply_tran(const std::string& keyword, std::vector<double>& data) const {
+    this->fp->apply_tran(keyword, data);
+}
+
+std::vector<char> FieldPropsManager::serialize_tran() const {
+    return this->fp->serialize_tran();
+}
+
+void FieldPropsManager::deserialize_tran(const std::vector<char>& buffer) {
+    this->fp->deserialize_tran(buffer);
+}
+
+bool FieldPropsManager::tran_active(const std::string& keyword) const {
+    return this->fp->tran_active(keyword);
+}
+
+template<class MapType>
+void apply_tran(const std::unordered_map<std::string, TranCalculator>& tran,
+                const MapType& double_data,
+                std::size_t active_size,
+                const std::string& keyword, std::vector<double>& data)
+{
+    const auto& calculator = tran.at(keyword);
+    for (const auto& action : calculator) {
+        const auto& action_data = double_data.at(action.field);
+
+        for (std::size_t index = 0; index < active_size; index++) {
+
+            if (action_data.value_status[index] != value::status::deck_value)
+                continue;
+
+            switch (action.op) {
+            case ScalarOperation::EQUAL:
+                data[index] = action_data.data[index];
+                break;
+
+            case ScalarOperation::MUL:
+                data[index] *= action_data.data[index];
+                break;
+
+            case ScalarOperation::MAX:
+                data[index] = std::max(action_data.data[index], data[index]);
```

Comment:
If `MAX` represents `MAXVALUE`, then this must be `std::min()`.

Reply:
@joakim-hove Aren't we missing operations here? Isn't there at least MINVALUE?

Reply:
Yes that is right - I just implemented the absolute bare minimum; we should `ScalarOperation::MIN` - could in principle also add `ScalarOperation::ADD`- but I guess addition of transmissibilities is seldom right.

Reply:
Fixed and added MINVALUE and ADD support.